docs/glossary.rst

```diff
@@ -0,0 +1,388 @@
+.. _glossary:
+
+========
+Glossary
+========
+
+This Reference article is a glossary. It is meant to provide a general overview
+of terms common to CrateDB Cloud. Because it forms a counterpart to the
+Glossary for CrateDB, we mainly cover terms here that are specific or primary
+to CrateDB Cloud; terms particular to the whole CrateDB architecture will
+appear in the CrateDB Glossary instead. We hope this Glossary will help you
+understand the basic meaning of concepts in the Console, Documentation, the
+CLI, and other Cloud-related sources. It is not meant to be a general guide to
+IT or IIoT terminology.
```

Comment:
I've so far always treated references to our specific repos/directories as proper nouns, as opposed to (say) 'documentation' in general. I don't really mind much either way, as long as we're consistent about it - maybe something we should decide on a standard for within the tech writing team?

Reply:
I think it depends on whether you're citing a full title or heading

so, for example, "the CrateDB Cloud Reference" or "the CrateDB Cloud How-To Guides" would be correct because those are the full titles

however, "the Reference" or "the How-To Guides" would not be correct, because those are not the full titles

if you don't want to use the full title, I think it's fine to use common nouns with regular sentence capitalization, e.g., "the reference documentation", "reference articles", "how-to guides", and so on

---

"Glossary" is an interesting edge case

as a section heading, the proper style is "Glossary", and if the heading itself is being cited (e.g., when referenced in a table of contents, admonition, or parenthetical), it should retain the capital letter

however, in most sentences, I imagine we will want to refer to "the glossary" using "glossary" as a common noun with regular sentence capitalization

Reply:
But why would it need to be the full title? If you refer to "the States", as shorthand for "the United States of America", as people often do, you would still capitalize the S in States. Similar with "the United Kingdom" (leaving off "of Great Britain and Northern Ireland").

Reply:
I guess based on the convo at Tech Writing Sync today, we will prioritize consistency!

And add more rules to the Vale linter!